pkg/api/rules-schema.json

```diff
@@ -192,6 +228,18 @@
           "items": {
             "$ref": "#/definitions/backend"
           }
+        },
+        "tcp_connect_timeout": {
+          "description": "Connect/read/write timeout when talking to a route. Default is 15. Unit is in seconds.",
+          "type": "number",
+          "minimum": 0,
+          "exclusiveMinimum": true
+        },
+        "max_retries": {
+          "description": "Maximum number of retries to connect to a route.  Defaults to 1.",
```

Comment:
http_req_retries
Number of retries for a given request.  The interval between retries will be determined automatically (25ms+). Actual number of retries attempted depends on the http_req_timeout